ddp-workspace/projects/ddp-dsm-ui/src/app/services/dsm.service.ts

```diff
@@ -611,6 +611,17 @@ export class DSMService {
     );
   }
 
+  public downloadParticipantFile( fileName: string, bucketName: string, blob: string, realm: string, mimeType: string ): Observable<any> {
+    const url = this.baseUrl + DSMService.UI + 'downloadFile';
+    const map: { name: string; value: any }[] = [];
+    map.push( {name: DSMService.REALM, value: realm} );
+    map.push( {name: 'fileName', value: fileName} );
+    map.push( {name: 'bucket', value: bucketName} );
+    map.push( {name: 'blob', value: blob} );
+    map.push( {name: 'responseType', value: 'blob'} );
```

Comment:
**_#optional_**
The line is unnecessary. 
That parameter is already in `buildQueryBlobHeader` method.